railties/lib/rails/generators/rails/scaffold_controller/USAGE

```diff
@@ -12,7 +12,7 @@ Description:
 Example:
     `rails generate scaffold_controller CreditCard`
 
-    Credit card controller with URLs like /credit_card/debit.
```

Comment:
Now only the URL is fixed.

Not a big deal but it doesn't feel right that the `scaffold_controller` USAGE mentions URLs (or routes) because the generator doesn't modify `config/routes.rb` - could that be a separate issue I should look into?